integration/fdd_aspnet_test.go

```diff
@@ -1,86 +0,0 @@
-package integration_test
```

Comment:
why don't we want these tests or fixtures anymore?

Reply:
It was not testing anything unique compared to the integration/framework_dependent_deployment_test.go test. And the test fixture was a 3.1 test fixture. Same for the integration/fde_aspnet_test.go